tests/tck/utils/assertions.py

```diff
@@ -1,10 +1,76 @@
+from numbers import Number
+from redisgraph import Node, Edge
 def is_number_tryexcept(s):
-    """ Returns True is string is a number. """
+    """ Returns True if string is a number. """
+    if not isinstance(s, (Number, basestring)):
+        return False
     try:
         float(s)
         return True
     except ValueError:
         return False
+    
+def prepareActualValue(actualValue):
+        actualValue = prepareString(actualValue)
```

Comment:
This seems like each function will overwrite the previous one,
my guess without reading the implementation of prepareString, stringToNumeric, nodeToString and edgeToString
is that they do nothing if actualValue is set.

Please avoid the unnecessary calls by returning once actualValue been set.

Reply:
fixed